### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yomiadetutu1**
Updated job titles

### CODE REVIEW

**Review by cwarnermm — DISMISSED** [review]
Thanks, @yomiadetutu1!

**Review by lynn915 — COMMENTED** [review]
Yomi, I don't think Natalie has changed her title yet? Can you confirm with her before making the change?

File: operations/workplace/people/README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ Description: >-
 
 | Name | Title |
 | :--- | :--- |
-| [Natalie Jew](https://community.mattermost.com/core/messages/natalie.jew) |VP, Human Resources|
+| [Natalie Jew](https://community.mattermost.com/core/messages/natalie.jew) |VP, People|
```
Comment (yomiadetutu1) [inline operations/workplace/people/README.md]:
```suggestion
| [Natalie Jew](https://community.mattermost.com/core/messages/natalie.jew) |VP, Human Resources|
```

Comment (yomiadetutu1) [inline operations/workplace/people/README.md]:
```suggestion
| [Natalie Jew](https://community.mattermost.com/core/messages/natalie.jew) |VP, Human Resources|
```

**Comment by yomiadetutu1** [comment]
Thanks @lynn915 , Natalie confirmed her title hasn't changed and I've reverted to the existing title.

File: operations/workplace/people/README.md (hunk 1 above)
Comment (yomiadetutu1) [reply operations/workplace/people/README.md]: Thanks @lynn915 , Natalie confirmed her title hasn't changed and I've reverted to the existing title.

**Review by lynn915 — APPROVED** [review]

**Pull request merged**